openupgrade_scripts/scripts/payment/18.0.2.0/noupdate_changes.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
+  <record id="payment_method_emi_india" model="payment.method">
+      <!-- manually added -->
+    <field name="code">emi_india</field>
+  </record>
```

Comment:
please do this in another file so that this doesn't keep showing up in the diff when a new analysis is generated. Also makes reviewing the automated PRs easier whenever that will be merged